.pre-commit-config.yaml

```diff
@@ -23,8 +23,6 @@ repos:
       - id: mdformat
         additional_dependencies:
           - mdformat-mkdocs
-          - mdformat-black
-          - black==23.1.0 # Must be the latest version of Black
```

Comment:
This is causing problems since we now have intentional non-Black-formatted snippets in the README. It seems like we'll run into this often now that we use our own formatter. I'm partial to removing, but also happy to add a file exclusion here if there's any hesitancy.